packages/core/src/Stores/client-store.js

```diff
@@ -2090,7 +2093,7 @@ export default class ClientStore extends BaseStore {
         if (response?.logout === 1) {
             this.cleanUp();
 
-            this.root_store.rudderstack.reset();
+            rudderstack.reset();
```

Comment:
@shayan-deriv this resets the `has_identified` to `false` when user logs out

Reply:
thanks man for the explanation. got it now :+1: